packages/react-examples/src/react-components/FluentUIComponents/Introduction.stories.mdx

```diff
@@ -1,9 +1,24 @@
 import { Meta } from '@storybook/addon-docs/blocks';
 
+<img
+src="https://user-images.githubusercontent.com/78806055/115746997-25872300-a395-11eb-8ad9-f486936f1bbb.png"
+style={{ width: '100%' }}
+/>
+
 <Meta title="Concepts/Introduction" />
 
-## @fluentui/react-components
+<center>
```

Comment:
**suggestion:** this is HTML4 era tag, not really if it works properly in HTML5 mode? 
you can replace this by following:


```diff
-<center>
+<div style={{textAlign:'center'}}>
```

Reply:
Thank you, I will try that out locally and test.

Reply:
Suggestion worked and the changes have been pushed.